test/dest.js

```diff
@@ -763,53 +763,6 @@ describe('dest stream', function() {
     stream.end();
   });
 
-  it('should write file atime and mtime using the vinyl stat', function(done) {
```

Comment:
There's another test [here](https://github.com/erikkemperman/vinyl-fs/blob/feature-stats/test/dest.js#L651) which does for just `mtime` what this one does for both `atime` and `mtime`.

Now the a file's `stat.atime` changes when its contents are read, and this is reflected on disk, there is no point in providing an expecting a fixed `atime` to result from a subsequent `statSync` on that file.

And so figured we'd only need the `mtime`-only version of this test.

But I am increasingly uncomfortable that we've got some of this stuff the wrong way around.